sdk/python/foundation-models/system/content-safety/deploy_with_content_safety.ipynb

```diff
@@ -0,0 +1,803 @@
+    "except Exception as ex:\n",
+    "    # NOTE: Update following workspace information to contain\n",
+    "    #       your subscription ID, resource group name, and workspace name\n",
+    "    client_config = {\n",
+    "        \"subscription_id\": subscription_id,\n",
+    "        \"resource_group\": resource_group,\n",
+    "        \"workspace_name\": workspace_name,\n",
+    "    }\n",
+    "    # write and reload from config file\n",
+    "    config_path = \"../.azureml/config.json\"\n",
+    "    os.makedirs(os.path.dirname(config_path), exist_ok=True)\n",
+    "    with open(config_path, \"w\") as fo:\n",
+    "        fo.write(json.dumps(client_config))\n",
+    "    ml_client = MLClient.from_config(credential=credential, path=config_path)\n",
+    "    \n",
+    "print(ml_client)\n",
+    "workspace_location = ml_client.workspaces.get(ml_client.workspace_name).location\n",
+    "subscription_id = ml_client.subscription_id\n",
+    "resource_group_name = ml_client.resource_group_name\n",
+    "workspace_name = ml_client.workspace_name\n",
+    "print(workspace_location)\n",
+    "print(subscription_id)\n",
+    "print(resource_group_name)\n",
+    "print(workspace_name)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4. Create Azure AI Content Safety"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 4.1 Choose a region for your Content Safety\n",
+    "Currently, Azure AI Content Safety only available in the following regions:\n",
+    "- East US\n",
+    "- West Europe\n",
+    "- Central US EUAP"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "aacs_location = \"east us\" # please choose the nearest location to your workspace\n",
+    "print(f\"will create aacs in {aacs_location}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.mgmt.cognitiveservices import CognitiveServicesManagementClient\n",
+    "from azure.identity import DefaultAzureCredential, InteractiveBrowserCredential\n",
+    "from azure.mgmt.cognitiveservices.models import Account, Sku, AccountProperties\n",
+    "import time\n",
+    "\n",
+    "try:\n",
+    "    credential = DefaultAzureCredential()\n",
+    "    # Check if given credential can get token successfully.\n",
+    "    credential.get_token(\"https://management.azure.com/.default\")\n",
+    "except Exception as ex:\n",
+    "    # Fall back to InteractiveBrowserCredential in case DefaultAzureCredential not work\n",
+    "    credential = InteractiveBrowserCredential()\n",
+    "\n",
+    "client = CognitiveServicesManagementClient(credential, subscription_id)\n",
+    "# create a new Cognitive Services Account\n",
+    "# name of the AACS to create\n",
+    "name_of_aacs = f\"{endpoint_name}-aacs\"\n",
```

Comment:
nit: Any ACS endpoint name restrictions? Length or character. I want to make sure just concatenating from `endpoint_name` will work

Reply:
This is just my preference. If we tell customers to make the `endpoint_name` to be globally unique, I think just use `endpoint_name` as ACS endpoint name should be OK, I will update it.

Reply:
The limits for AACS name are: "Only alphanumeric characters and hyphens are allowed. The value must be 2-64 characters long and cannot start or end with a hyphen." If the endpoint_name provided by the customer is too long, should we perform substring truncation or should we set a maximum length limit of 59 characters (64 - 5) for the endpoint_name?

Reply:
The length limit for the endpoint name is 32 characters, while the AACS limit is 64 characters. Therefore, concatenating from `endpoint_name` will not be a problem.